### PR TITLE
Bookstore: initialize project with components

### PR DESCRIPTION
**Pull request by rabayarabu**
- Create components inside of the /components directory:
- Contains individual book state, as well as a button to delete the book
- Contains a list to renders individual books
- Contains a form to create new books
- Contains a navigation to link to the 2 routes you created

### CODE REVIEW

**Comment by prolajumokeoni** [comment]
Hi @rabayarabu, Kindly add a professional and descriptive readme

**Review by m05e5 — CHANGES_REQUESTED** [review]
Hi @rabayarabu , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't have a professional readme file. Kindly edit your readme file and make sure it follows Microverse readme requirements

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+import Navbar from './components/Navbar';
+
+function App() {
+  return (
+    <div className="App">
+      <Router>
+        <Navbar />
+        <Routes>
+          <Route path="/" element={<Bookstore />}>Home</Route>
+          <Route path="/category" element={<Bookstore />}>Home</Route>
```
Comment (CollinsTatang) [inline src/App.js:13]: ### **Optional**

- [ ] Kindly create and add a Categories component to your routes to make it more explicit.

**Review by CollinsTatang — APPROVED** [review]
# Approved 👏  
### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**